### ISSUE TITLE
[tx] -dcf mode fails to dump hinted CFF2 variable font

### ISSUE THREAD

**Issue by blueshade7** [adobe-type-tools/afdko#322]
Run tx as:
```
tx -dcf KozSmaGoPr6NTestHints.otf
```
Result:
```
tx: Type 2 stack underflow
tx: fatal error
### Header (00000cb8-00000cbc)
major  =2
minor  =0
hdrSize=5
offSize=0
```

**Comment by blueshade7** [comment]
[KozSmaGoPr6NTestHints.otf.zip](https://github.com/adobe-type-tools/afdko/files/1929308/KozSmaGoPr6NTestHints.otf.zip)

**Comment by miguelsousa** [comment]
Fixed by #323
